### DISCUSSION TITLE
chore: find a better way to track TODOs in code

### DISCUSSION THREAD

**Discussion by mohanraj-r** [salesforce/sa11y#57] [Tech Debt]
> Trevor: Should we capture these TODOs as issues?

Thanks Trevor, Yes that is a good idea. 
* I have been thinking of exploring [GitHub Discussions](https://docs.github.com/en/discussions) to capture TODOs so that Issues are focussed on bugs etc. What do you think about that?
* Looks like there are also tools to convert TODOs in code to issues automatically e.g. [TODO to Issue · Actions · GitHub Marketplace · GitHub](https://github.com/marketplace/actions/todo-to-issue). I will explore them more.

_Originally posted by @mohanraj-r in https://github.com/salesforce/sa11y/pull/54#r583933982_

**Comment by trevor-bliss** [comment]
I personally think we should keep things simple to start. There's already a significant amount of tooling used in this repo (see dev dependencies in package.json). I know you've been doing a good job at staying on top of keeping them up to date, but each new one is something else to learn for new users.

How about just starting with filing an issue for the TODO comment and linking to that issue. 

If it becomes hard to scale then we can explore some more advanced tooling.

```
// TODO [#43]: do XYZ here after issue is fixed
```
